### PR TITLE
Updated the colour of profile badge and fixed errors in the same file

### PR DESCRIPTION
**Pull request by Parthan-Akon**
The user profile has got errors which has been fixed.
image

Please refer the image attached.

### CODE REVIEW

**Comment by nihal467** [comment]
@Parthan-Sachin @gigincg is it good for testing

File: src/Components/Users/UserProfile.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ const editFormReducer = (state: State, action: Action) => {
 export default function UserProfile() {
   const [states, dispatch] = useReducer(editFormReducer, initialState);
 
-  const state: any = useSelector((state) => state);
+  const state: any = useSelector((state1) => state1);
```
Comment (gigincg) [inline src/Components/Users/UserProfile.tsx:91]: This is unnecessary

File: src/Components/Users/UserProfile.tsx [hunk 2]
```diff
@@ -139,12 +139,14 @@ export default function UserProfile() {
   );
 
   const validateForm = () => {
-    let errors = { ...initError };
+    const errors = { ...initError };
```
Comment (gigincg) [inline src/Components/Users/UserProfile.tsx:142]: errors should be declared as `let` since it's mutated later in this function

**Review by gigincg — CHANGES_REQUESTED** [review]
Given the merge conflict and the issues in the PR, please make a separate PR

**Pull request closed without merging**